zone/zonedb.cpp

```diff
@@ -354,8 +357,8 @@ std::unique_ptr<EQ::ItemInstance> ZoneDatabase::LoadSingleTraderItem(uint32 char
 	}
 
 	inst->SetCharges(charges);
-	inst->SetSerialNumber(serial_number);
-	inst->SetMerchantSlot(serial_number);
+	inst->SetUniqueID(serial_number);
+	//FIX inst->SetMerchantSlot(serial_number);
 	inst->SetPrice(cost);
 
```

Comment:
Line 386 will need changed:

From
`"`char_id` = '{}' AND `item_id` = {}",` 

To:
`"`character_id` = '{}' AND `item_id` = {}",`

Reply:
Updated these.  Found a few.